src/CommandingOfficers/AW4/RuinedCommander.java

```diff
@@ -168,7 +168,7 @@ public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCo
     if( !isInZone(chargeSource) )
       return 0;
 
-    int guiHPDealt = enemy.getHealthDamage() / 10;
+    int guiHPDealt =  enemy.getHPDamage();
```

Comment:
Someone doesn't have auto-formatting enabled. Me. I don't. But also you don't.

Reply:
It's formatted to align with all the other statements with the same meaning across other CO classes.

...and yeah, I've given up on compulsive auto-format because I'm a vertical alignment gremlin.